myapp/config/locales/views/tasks/ja.yml

```diff
@@ -11,9 +11,17 @@ ja:
     show: '詳細'
     back: '戻る'
 
+  form:
+    unselected: '--未選択--'
```

Comment:
It's an unused definition, so could you delete it ?
ref: https://github.com/Fablic/training/pull/1520/commits/b8ad578725a1a0429bf54491b6463ba3c146a1b4